log/logrus/logrus_logger.go

```diff
@@ -10,18 +10,38 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-type logrusLogger struct {
+type Logger struct {
 	logrus.FieldLogger
+	logrus.Level
 }
 
+type Option func(*Logger)
+
 var errMissingValue = errors.New("(MISSING)")
 
-// NewLogrusLogger returns a go-kit log.Logger that sends log events to a Logrus logger.
-func NewLogrusLogger(logger logrus.FieldLogger) log.Logger {
-	return &logrusLogger{logger}
+// NewLogger returns a go-kit log.Logger that sends log events to a Logrus logger.
+func NewLogger(logger logrus.FieldLogger, options ...Option) log.Logger {
+	l := &Logger{
+		FieldLogger: logger,
+		Level:       logrus.InfoLevel,
+	}
+
+	for _, optFunc := range options {
+		optFunc(l)
+	}
+
+	return l
 }
 
-func (l logrusLogger) Log(keyvals ...interface{}) error {
+// WithLevel configures a logrus logger to set specific log
+// level to log messages with
```

Comment:
This comment starts out ok, but seems to leave something dangling at the end. How about this:

```
// WithLevel configures a logrus logger to log at level for all events.
```